pkg/extensions/sync/utils.go

```diff
@@ -45,12 +45,14 @@ func getTagFromRef(ref types.ImageReference, log log.Logger) reference.Tagged {
 }
 
 // getRepoFromRef returns repo name from a registry ImageReference.
+/* Not used anymore
```

Comment:
No! Remove any unused dead code pls. How come lint is not catching this.

Reply:
Oops! Must have forgot that there. Removed it.